controllers/timechaos/types.go

```diff
@@ -119,7 +119,7 @@ func (r *Reconciler) Apply(ctx context.Context, req ctrl.Request, chaos reconcil
 	return nil
 }
 
-// Recover means the reonciler recover the chaos action
+// Recover means the reconciler recover the chaos action
```

Comment:
Umm, should this be `reconciler recovers`?

Reply:
done